packages/flutter/test/foundation/stack_frame_test.dart

```diff
@@ -68,7 +68,7 @@ void main() {
       expect(frames.contains(StackFrame.stackOverFlowElision), true);
     }
     expect(overflowed, true);
-  }, skip: isBrowser);
+  }, skip: isBrowser); // Reference needed
```

Comment:
@dnfield is there an issue or comment I can add here?

It looks like you left yourself a note in the change that added this (#49629) 🤣  : 

> For future me: The browser test harness doesn't seem to be able to handle this, but the VM does.

Reply:
hah, sorry about that.  I think linking this to the PR is probably ok.

Unfortunately, the browser harness can't handle testing something that intentionally causes a stackoverflow - the whole harness just crashes.  @yjbanov @ferhatb @jonahwilliams might know if anything can be done about that (or know who would know).

Reply:
Yes, the browser completely freezes in that case and never responds back.

Reply:
So maybe just

> // The VM test harness can handle a stack overflow, but the browser cannot - running this test in a browser will cause it to become unresponsive.